application/frontend/config/main.php

```diff
@@ -30,14 +33,18 @@
             // http://www.yiiframework.com/doc-2.0/guide-rest-routing.html
             'rules' => [
                 [
+                    // http://www.yiiframework.com/doc-2.0/yii-rest-urlrule.html
                     'class' => 'yii\rest\UrlRule',
-                    'controller' => ['user', 'authentication'],
+                    'controller' => ['authentication', 'user'],
+                    'extraPatterns' => [
```

Comment:
Will this `extraPatterns` entry apply to both the `authentication` and `user` controllers? I'm not sure I understand the way this is supposed to be used (specifically the patterns in this list, since I would have expected them to start with `/user` or `/authentication`).